GW2EIBuilders/Resources/htmlTemplates/tmplSimpleRotation.html

```diff
@@ -48,52 +69,32 @@
                 Skills <span class="caret"></span>
             </a>
             <ul class="dropdown-menu p-2 font-weight-normal scrollabledropdown" style="min-width: 165px; max-width: 165px;">
-                <div class="d-flex flex-column justify-content-center align-items-center mb-1"
```

Comment:
This one was giving issues, visual studio was recognising it as unused div that has no end tag which made everything get messed up